requirements.txt

```diff
@@ -1,9 +1,16 @@
-###############################################################
 #
-# Add all python packages that dont depend on C libraries here
+# scientific packages
+#
+Cython>=0.24
+nimfa==1.2.3
+numpy>=1.10.2
+pandas>=0.18.1
+scikit-image>=0.12.3
+scikit-learn>=0.17.1
+scipy>=0.16.0
+
+#
+# dask packages
 #
-###############################################################
-lxml>=3.4.4
-docker-py==1.9.0
```

Comment:
Otherwise, we probably just need to check to make sure they work with the current latest.